src/datasource_application.ts

```diff
@@ -11,6 +11,8 @@ import _ from 'lodash';
 
 export default class InstanaApplicationDataSource extends AbstractDatasource {
   applicationsCache: Cache<Promise<Array<Selectable>>>;
+  queryInvervallAppCallsLimit: number;
```

Comment:
typo in variable names.. interval only with one l in English, used various times with two ll